float32 servo_1
float32 servo_2
